cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/utils/GroupedMessageEntityUtils.java

```diff
@@ -38,24 +39,27 @@ public class GroupedMessageEntityUtils {
 
     private static final Logger logger = LoggerFactory.getLogger(GroupedMessageEntityUtils.class);
 
-    public static SerializedEntity<GroupedMessageBatchEntity> toSerializedEntity (GroupedMessageBatchToStore batch, PageId pageId, int maxUncompressedSize)
-            throws IOException	{
+    public static SerializedEntity<GroupedMessageBatchEntity> toSerializedEntity (GroupedMessageBatchToStore batch,
+                                                                                  PageId pageId,
+                                                                                  CompressionType compressionType,
+                                                                                  int maxUncompressedSize)
+            throws IOException, CompressException {
```

Comment:
I think we should correct the formatting here. It is quite unreadable (I understand that it is readable in IDE but here it is not)
```suggestion
    public static SerializedEntity<GroupedMessageBatchEntity> toSerializedEntity(
            GroupedMessageBatchToStore batch,
            PageId pageId,
            CompressionType compressionType,
            int maxUncompressedSize
    ) throws IOException, CompressException {
```